model_hub/examples/huggingface/token-classification/ner_utils.py

```diff
@@ -0,0 +1,128 @@
+"""
+These functions extract parts of the code from the named-entity recognition example
+in huggingface transformers.  The source example code changes rapidly due to frequent refactorings
+but the main structure should be similar.
+See the original example at:
+https://github.com/huggingface/transformers/blob/master/examples/token-classification/run_ner.py
+
+Note this code is separated from ner_trial.py since it is not determined specific code and
+is necessary even if implementing outside of determined.
+"""
+from typing import Any, Dict, List
+
+import datasets
+import numpy as np
+from attrdict import AttrDict
+from model_hub import huggingface as hf
```

Comment:
normally we distinguish local packages from third-party packages.  I think you may have to configure `isort` so that it does this automatically.